wikimedia/wikipedia-ios
# Add app install ID to EditAttemptStep

**Phabricator:**
https://phabricator.wikimedia.org/T358735

### Notes
This PR logs app_install_id as a part of our EditAttemptStep metrics.

### Test Steps
1. Go to an article, tap overflow menu > "Edit source"
2. Watch the console and confirm you see an EditAttemptStep payload. Confirm you see a populated "app_install_id" key in the event.
3. (Optional) Confirm you see the same payload post when proxying, and the response is successful.
